releasing/lagom.md

```diff
@@ -120,21 +120,44 @@ Merge your release pull request and deploy the updated website from `vegemite`.
 *NOTE*: the command logs its output to `~/logs/lagom-nightly-deploy-master-1532020306.log` or similar. You need
 to find the most recent file to `tail` if you can to see what is happening.
 
-## Update the g8 templates and example projects
+## Update the downstream projects and other deps
+
+#### Downstream
+
+* https://github.com/playframework/play-grpc/blob/master/project/Dependencies.scala#L14
```

Comment:
Should we also include https://github.com/akka/akka-persistence-couchbase/blob/master/project/Dependencies.scala#L14?

Reply:
Aaah! Good call!